pandapower/opf/run_powermodels.py

```diff
@@ -33,29 +34,36 @@ def _runpm(net, delete_buffer_file=True):  # pragma: no cover
     # writes pm json to disk, which is loaded afterwards in julia
     buffer_file = dump_pm_json(pm)
     # run power models optimization in julia
-    result_pm = _call_powermodels(buffer_file, net._options["julia_file"])
+    result_pm = _call_powermodels(buffer_file, net._options["julia_file"], sysimage_file)
     # read results and write back to net
     read_pm_results_to_net(net, ppc, ppci, result_pm)
     if delete_buffer_file:
         # delete buffer file after calculation
         os.remove(buffer_file)
```

Comment:
if this is the only "os" function call now, it would be nice to replace it with `pathlib.Path.unlink()`